### DISCUSSION TITLE
How to solve External Library failing to scan when using SMB and a Synology NAS

### DISCUSSION THREAD

**Discussion by elfensky** [immich-app/immich#13864] [Community Guides]
TLDR: add `**/#recycle/**` as an exclusion pattern

<img width="2301" alt="Screenshot 2024-10-31 at 23 16 47" src="https://github.com/user-attachments/assets/93225674-5318-4d85-9f6d-2a7290bb68f0">

<img width="504" alt="image" src="https://github.com/user-attachments/assets/428ff7a1-b333-4fd6-8cbf-a482e6263743">

When you have the recycle bin feature enabled on Synology, it is usually protected by the admin user. Assuming you make a separate user for your immich container, that user likely won't be able to access the recycle bin folder. 

Due to the hashtag, it's the first one in the list, and instead of continuing trying to process the other folders, Immich simply stops and fails. Whether that should be changed is debatable, but the solution is simple - stop it from trying to access it.
